spec/02-integration/07-sdk/03-cluster_spec.lua

```diff
@@ -40,8 +40,9 @@ fixtures_cp.http_mock.my_server_block = [[
   }
 ]]
 
--- TODO: reenable the inc sync test
-for _, inc_sync in ipairs { "off"  } do
+for _, v in ipairs({ {"off", "off"}, {"on", "off"}, {"on", "on"}, }) do
```

Comment:
How about making it a part of helpers:

```suggestion
for _, v in helpers.all_rpc_combination() do
```

Reply:
Nice idea, perhaps we could do it in another PR.